### PR TITLE
Add tamplate to home view

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/assets/stylesheets/main.css [hunk 1]
```diff
@@ -109,6 +109,64 @@ body {
   color: #7d8a93;
 }
 
+.logo-left {
+
```
Comment (ozimeu) [inline app/assets/stylesheets/main.css]: empty class

File: app/views/layouts/application.haml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    %meta{:content => "", :name => "description"}/
+    %meta{:content => "width=device-width, initial-scale=1", :name => "viewport"}/
+    %title Melange
+    / Place favicon.ico in the root directory
+    %link{:href => "public/favicon.ico", :rel => "shortcut icon"}/
+    / Mandatory Stylesheets
+    %link{:href => "css/base/bootstrap.css", :rel => "stylesheet"}/
+    %link{:href => "https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800", :rel => "stylesheet"}/
+    / Plugin Stylesheets
+    %link{:href => "css/plugins/owl-carousel/owl.carousel.min.css", :rel => "stylesheet"}/
```
Comment (ozimeu) [inline app/views/layouts/application.haml]: css files are loaded by the `stylesheet link tag` - you should not include them here

File: app/views/layouts/application.haml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    %link{:href => "css/custom.css", :rel => "stylesheet"}/
+
+    
+    / Menu
+    %header.tp--header.tp--header-fixed.tp--header-shrink.tp--header-1
+      %nav.navbar-wrapper
+        .container.clearfix
+          %h1.logo
+            = image_tag 'menu/logo.png', class: 'logo-left tp--scroll'
+            %a.tp--scroll{:href => "#intro"}
```
Comment (ozimeu) [inline app/views/layouts/application.haml]: old hash notation, no spaces between brackets

File: app/views/layouts/application.haml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+              %img.logo-main{:alt => "Melange", :src => "images/logo.png"}/
+              %img.logo-dark{:alt => "Melange", :src => "images/logo.png"}/
+            
+          = render "shared/nav"
+  %body
+    = yield
```
Comment (ozimeu) [inline app/views/layouts/application.haml]: missing empty line at the end of the file

File: app/views/layouts/application.haml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    
+    / Menu
+    %header.tp--header.tp--header-fixed.tp--header-shrink.tp--header-1
+      %nav.navbar-wrapper
+        .container.clearfix
+          %h1.logo
+            = image_tag 'menu/logo.png', class: 'logo-left tp--scroll'
+            %a.tp--scroll{:href => "#intro"}
+              %img.logo-main{:alt => "Melange", :src => "images/logo.png"}/
+              %img.logo-dark{:alt => "Melange", :src => "images/logo.png"}/
```
Comment (ozimeu) [inline app/views/layouts/application.haml]: ☝️

File: app/views/shared/_nav.haml [hunk 6]
```diff
@@ -1,12 +1,19 @@
...
-- if current_user 
-    = link_to "Wyloguj", destroy_user_session_path, method: :delete 
-- else 
-= link_to "Zaloguj", new_user_session_path 
\ No newline at end of file
+.navbar.pull-right
+    .navbar-collapse.collapse
+        %ul.nav.navbar-nav
+            %li
+                %button.tp--scroll.margin-button{:href => "#intro"} Domówka
```
Comment (ozimeu) [inline app/views/shared/_nav.haml]: spaces `{` and old hash rocket notation

File: app/views/shared/_nav.haml [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} Staty
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} O co kaman?
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} Kontakt
+            %li.tp--nav-divider
+                %a
+            %li.tp--nav-signup
+                %button.sing-in Cho z nami!
```
Comment (ozimeu) [inline app/views/shared/_nav.haml]: missing empty line at the end

File: app/views/shared/_nav.haml [hunk 8]
```diff
@@ -1,12 +1,19 @@
...
+            %li
+                %button.tp--scroll.margin-button{:href => "#about"} Melanżowcy
+            %li
+                %button.tp--scroll.margin-button{:href => "#projects"} Gdzie?
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} Staty
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} O co kaman?
+            %li
+                %button.tp--scroll.margin-button{:href => "#team"} Kontakt
```
Comment (ozimeu) [inline app/views/shared/_nav.haml]: ☝️

File: app/views/static_pages/index.haml [hunk 9]
```diff
@@ -1,473 +1,106 @@
...
-              %i.icon-plus
-              Preview
-            %a.link{:href => "#"}
-              %i.icon-link
-              Website
-/
-  Team
-  \==================================================
+          %h3 Color Party
+
```
Comment (ozimeu) [inline app/views/static_pages/index.haml:62]: extra empty line

**Review by QuaternionX — APPROVED** [review]

**Pull request merged**